### PR TITLE
adds cicd for repo

### PR DESCRIPTION
**Pull request by qjflores**
adds cicd to repo

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package-lock = false
+@themuse:registry=https://verdaccio.inf.themuse.com/
```
Comment (qjflores) [inline .npmrc:2]: this is needed to get the dependencies from verdaccio, such as in `package.json`, `@themuse/conflator`

File: package.json [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+  },
   "devDependencies": {
     "ava": "~1.2.0",
+    "codecov": "~3.1.0",
+    "nyc": "~13.1.0",
     "standard": "~12.0.1"
   },
   "dependencies": {
-    "conflator": "git+ssh://git@github.com/dailymuse/conflator.git#develop",
+    "@themuse/conflator": "~0.0.1",
```
Comment (qjflores) [inline package.json:18]: consumes `conflator` from verdaccio

File: src/electrojector.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const path = require('path')
-const Conflator = require('conflator')
+const Conflator = require('@themuse/conflator')
```
Comment (qjflores) [inline src/electrojector.js:2]: without `@themuse` we'll get an error

File: src/electrojector.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ class Electrojector {
```
Comment (qjflores) [inline src/electrojector.js:14]: the rest of the changes in this file are from the linter

File: test/core.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ const sinon = require('sinon')
 const electrojector = require('../index')
```
Comment (qjflores) [inline test/core.js:4]: the rest of the changes in this file are from the linter

File: test/dependencies/dependent.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
 module.exports = deps => {
```
Comment (qjflores) [inline test/dependencies/dependent.js:1]: the rest of the changes in this file are from the linter

File: test/node.js [hunk 7]
```diff
@@ -3,71 +3,71 @@ const electrojector = require('../index')
 const path = require('path')
```
Comment (qjflores) [inline test/node.js:4]: the rest of the changes in this file are from the linter

File: .npmrc (hunk 1 above)
Comment (davidkaminsky) [reply .npmrc:2]: Cool

**Review by davidkaminsky — APPROVED** [review]

**Pull request merged**